sms.js

```diff
@@ -0,0 +1,11 @@
+const accountSid = "AC6e4a0c264045033cb9dd3df126bedbcc";
+const authToken = "42085f49dc665abd66771ea32795793d"; 
+const client = require('twilio')(accountSid, authToken);
```

Comment:
How would you do if you have send the same message to a list of customers and customize the text message so it includes the name of the customer?

Let's say your customers are structured this way
```
const customers = [
  { id: 1, name: 'Lee', phone_number: '1111111111' },
  { id: 2, name: 'Kalan', phone_number: '1111111111' },
  { id: 3, name: 'Hollis', phone_number: '1111111111' }
]
```

So the final result is something like:
- `Lee use the coupon FREESHIPPING during your next purchase!`
- `Kalan use the coupon FREESHIPPING during your next purchase!`
- `Hollis use the coupon FREESHIPPING during your next purchase!`